### PR TITLE
[RNMobile] hide "⤴️" button from FloatingToolbar for root list blocks

### PR DESCRIPTION
**Pull request by jbinda**
## Description

According to [this issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1967) and Slack discussion with Thomas this PR changes hides the ⤴️ button on the FloatingToolbar for top-level blocks.

Please also refer to:
[Related gutenberg-mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/1973)
[Related gutenberg-mobile issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1967)

## How has this been tested?

1. Start with below code in `initial-html`

<details>

```
/**
 * @format
 * @flow
 */

export default `
<!-- wp:group -->
<div class="wp-block-group"><div class="wp-block-group__inner-container"><!-- wp:image {"id":1,"sizeSlug":"large"} -->
<figure class="wp-block-image size-large"><img src="https://cldup.com/cXyG__fTLN.jpg" alt="" class="wp-image-1"/></figure>
<!-- /wp:image -->

<!-- wp:media-text -->
<div class="wp-block-media-text is-stacked-on-mobile"><figure class="wp-block-media-text__media"></figure><div class="wp-block-media-text__content"><!-- wp:paragraph -->
<p></p>
<!-- /wp:paragraph --></div></div>
<!-- /wp:media-text -->

<!-- wp:group -->
<div class="wp-block-group"><div class="wp-block-group__inner-container"><!-- wp:image -->
<figure class="wp-block-image"><img alt=""/></figure>
<!-- /wp:image --></div></div>
<!-- /wp:group -->

<!-- wp:video {"id":2} -->
<figure class="wp-block-video"><video controls src="https://i.cloudup.com/YtZFJbuQCE.mov"></video></figure>
<!-- /wp:video -->

<!-- wp:gallery {"ids":[1,3]} -->
<figure class="wp-block-gallery columns-2 is-cropped"><ul class="blocks-gallery-grid"><li class="blocks-gallery-item"><figure><img src="https://cldup.com/cXyG__fTLN.jpg" data-id="1" class="wp-image-1"/></figure></li><li class="blocks-gallery-item"><figure><img src="https://cldup.com/cXyG__fTLN.jpg" data-id="3" class="wp-image-3"/></figure></li></ul></figure>
<!-- /wp:gallery --></div></div>
<!-- /wp:group -->


`;
```

</details>

2. Change selection between blocks
3. Observe that the ⤴️ button and the pipe after is hidden when top-most block is selected

## Screenshots 

<image src="https://user-images.githubusercontent.com/21242757/75545804-8f24cf00-5a27-11ea-869c-47ed8c9c938d.gif" width="300" />

## Types of changes

New feature - affects only design of FloatingToolbar

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/block-editor/src/components/block-list/block.native.js [hunk 1]
```diff
@@ -206,16 +206,18 @@ class BlockListBlock extends Component {
...
-								<ToolbarButton
-									title={ __( 'Navigate Up' ) }
-									onClick={ () =>
-										this.props.onSelect( parentId )
-									}
-									icon={ NavigateUpSVG }
-								/>
-								<View style={ styles.pipe } />
-							</Toolbar>
+							{ this.props.hasParent && (
```
Comment (dratwas) [inline packages/block-editor/src/components/block-list/block.native.js]: I suggest adding `hasParent` to the destructuring assignment like the rest of the props.

File: packages/block-editor/src/components/block-list/block.native.js [hunk 2]
```diff
@@ -206,16 +206,18 @@ class BlockListBlock extends Component {
...
+								<Toolbar passedStyle={ styles.toolbar }>
+									<ToolbarButton
+										title={ __( 'Navigate Up' ) }
+										onClick={ () =>
+											this.props.onSelect( parentId )
```
Comment (dratwas) [inline packages/block-editor/src/components/block-list/block.native.js]: Same here. I know it was like this before, but if you could change that :)

**Review by dratwas — APPROVED** [review]
I left 2 comments, but they are not a blocker :) Seems good!

File: packages/block-editor/src/components/block-list/block.native.js (hunk 2 above)
Comment (jbinda) [reply packages/block-editor/src/components/block-list/block.native.js]: done !

File: packages/block-editor/src/components/block-list/block.native.js (hunk 1 above)
Comment (jbinda) [reply packages/block-editor/src/components/block-list/block.native.js]: done !

**Pull request merged**